src/olympia/addons/tests/test_utils_.py

```diff
@@ -27,41 +29,70 @@
 
 
 @pytest.mark.django_db
-@pytest.mark.parametrize('name, allowed, email', (
+@pytest.mark.parametrize('name, allowed, email, content_optmzn_waffle', (
```

Comment:
`content_optmzn_waffle` is a weird name...

Reply:
yeah... I was just trying to choose a name short enough to fit it within the line limit.  It'll be gone again in a few months after the changes launch.